jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -12,6 +12,11 @@ pipeline {
             description: 'Input manifest under the manifests folder, e.g. 2.0.0/opensearch-2.0.0.yml.',
             trim: true
         )
+        string(
```

Comment:
I wonder do we want this parameter and each distribution build would be specific to one distribution? or we run build for both tarball and rpm in parallel within one job.